themes/Backend/ExtJs/backend/order/model/order.js

```diff
@@ -162,7 +163,8 @@ Ext.define('Shopware.apps.Order.model.Order', {
          */
         reader: {
             type: 'json',
-            root: 'data'
+            root: 'data',
+            messageProperty: 'message',
```

Comment:
We need to be consistent with Shopware's code style here, so please remove the trailing comma.